domains/gabivlg.json

```diff
@@ -0,0 +1,12 @@
+{
+        "owner": {
+           "username": "GabiVlg",
+           "email": "gabivoinea111111@yahoo.com",
+           "discord": "368732292408934403"
+        },
+    
+        "record": {
+            "CNAME": "gabivlg.is-a.dev 10800 in cname domain.par.clever-cloud.com."
```

Comment:
Firstly, your CNAME has to actually point somewhere, not to the domain you want. Secondly, your CNAME cannot have anything except the domain name itself, e.g. `orangc.xyz`, not `orangc.xyz uwu owo in cname`.

Reply:
gabivlg CNAME www.gabivlg.is-a.dev.cdn.cloudflare.net